<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KAlgorithmAnalysisClass</class>
 <widget class="QWidget" name="KAlgorithmAnalysisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KAlgorithmAnalysis</string>
  </property>
  <widget class="QLineEdit" name="vectorFileInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>输入矢量：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openVectorFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="AoGIS.qrc">
     <normaloff>:/AoGIS/res/openFile.png</normaloff>:/AoGIS/res/openFile.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="vectorFileOutput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>输出矢量：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveVectorFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="AoGIS.qrc">
     <normaloff>:/AoGIS/res/saveFile.png</normaloff>:/AoGIS/res/saveFile.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="begin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AoGIS.qrc"/>
 </resources>
 <connections/>
</ui>
